### ISSUE TITLE
 _repr_html_ creates extra scrollbar in ipython notebook

### ISSUE THREAD

**Issue by xLaszlo** [pydata/pandas#9268]
This line: div style="max-height:1000px;max-width:1500px;overflow:auto;" is added to the output if the df is too large. This behaviour is handled in ipython anyway so should be removed or optional.

**Comment by jreback** [comment]
pls show a picture of what you mean.

**Comment by TomAugspurger** [comment]
I think the OP is talking about if you click on the area that scrolls the output.

![image](https://cloud.githubusercontent.com/assets/1312546/5778858/e96d4b44-9d63-11e4-872b-921f33bccb75.png)

Doesn't make sense to remove our truncation/scrolling though. IPython's default is to display the whole output by default. It could be made as an option I suppose.

**Comment by xLaszlo** [comment]
That's exactly what I meant, thanks for posting.
An option would be fine as well.

**Comment by kpweiler** [comment]
I was about to add a feature request to make expose this max-height and max-width (of the frame) so that I could make it larger. If you want to look at a dataframe in iPython with many columns on a large screen, this option would be very useful.

**Comment by TomAugspurger** [comment]
@kpweiler we do have those options already, if I'm understanding correctly.

Have a a look at http://pandas.pydata.org/pandas-docs/dev/generated/pandas.set_option.html

specifically `display.max_columns` and `display.max_rows`.

@xLaszlo do you want to check on how this behaves with IPython 3.0 and 3.1? I think they made some changes w/ how scrolling is handled there. A PR making our styling optional would be great.

**Comment by kpweiler** [comment]
@TomAugspurger if you look at the function, _repr_html_ in core/Frame.py, you will notice that it returns hardcoded values for max-height and max-width. The display.max_columns and display.max_rows certainly allow all the columns and rows to be displayed, it just arbitrarily cuts them off with a scrollbar at 1000px and 1500px respectively. For the time being, I've just hacked at the pandas code to return max-width:none, but obviously, if I upgrade that will go away.

I have changed my ipython notebook custom.css to have an infinitely resizable (in width) notebook, which I would like the pandas dataframe to follow.
